src/SearchBarForm.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+class SearchBarForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      title: ''
+
+    };
+  }
+
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const { title } = this.state;
+    this.props.handleSearch(title);
+    this.handleReset();
+  //  console.log(title);
+  }
+
+  handleReset() {
+    this.setState({
+      title: []
```

Comment:
I know you're still working on this, looking really good!

Probably want this to be `title: ''` not an empty array